File: app/src/main/java/com/mmjang/ankihelper/data/dict/Oalde10Entry.java
package com.mmjang.ankihelper.data.dict;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "dict")
public class Oalde10Entry {
    @PrimaryKey
    public String hwd;

    public String phrase;

    public String sense;

    public String phonetics;

    public String defEn;

    public String defCn;
}

File: app/src/main/java/com/lmyby/ankiquicker/data/dict/Oalde10Entry.kt
package com.lmyby.ankiquicker.data.dict

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Converted to Kotlin as part of Phase 2 data model migration
 */
@Entity(tableName = "dict")
data class Oalde10Entry(
    @PrimaryKey
    var hwd: String = "",
    var phrase: String? = null,
    var sense: String? = null,
    var phonetics: String? = null,
    var defEn: String? = null,
    var defCn: String? = null
)
